dist/index.js

```diff
@@ -27,7 +27,8 @@ exports.githubConfig = {
     branchdir: process.env.GITHUB_WORKSPACE ? process.env.GITHUB_WORKSPACE : '',
     eventName: process.env.GITHUB_EVENT_NAME ? process.env.GITHUB_EVENT_NAME : '',
     runnerOS: process.env.RUNNER_OS ? process.env.RUNNER_OS : '',
-    pullRequestNumber: process.env.PULL_REQUEST_NUMBER ? process.env.PULL_REQUEST_NUMBER : pullRequestNumber
+    pullRequestNumber: process.env.PULL_REQUEST_NUMBER ? process.env.PULL_REQUEST_NUMBER : pullRequestNumber,
+    debugger: process.env.ACTIONS_RUNNER_DEBUG
```

Comment:
I guess this goes well, as when debug is not set, this variable will give a 'falsy' value.
However, if I (for whatever reason), set it to '0' or 'false', I guess debugging will be enabled?

Reply:
Will fix this with the inbuilt core function isDebug()